test/integration/default/controls/_mapdata_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Replace per minion strings
+replacement = {
+  hostname: system.hostname
+}
```

Comment:
Should this be removed since it's not actually being used in this formula?

Reply:
It's not used and does nothing. I was wondering if we should let this as an example.